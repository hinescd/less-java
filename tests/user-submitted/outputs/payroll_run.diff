--- tests/user-submitted/outputs/payroll_run.out	2019-02-04 13:12:57.048133989 -0500
+++ tests/user-submitted/outputs/payroll_run.exp	2019-02-04 13:12:57.195135909 -0500
@@ -1,3 +1,3 @@
 What is your name? How many hours did you work this week? What is your hourly payrate? 
-Hello Mike
-Your gross pay is $530.000000
+Hello Charlie
+Your gross pay is $10500.000000
