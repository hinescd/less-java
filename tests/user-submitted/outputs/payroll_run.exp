What is your name? How many hours did you work this week? What is your hourly payrate? 
Hello Charlie
Your gross pay is $10500.000000
