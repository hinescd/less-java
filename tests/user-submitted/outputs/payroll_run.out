What is your name? How many hours did you work this week? What is your hourly payrate? 
Hello Mike
Your gross pay is $530.000000
